spec/Powertrain/CombustionEngine.vspec

```diff
@@ -169,11 +222,40 @@
   max: 3000
   description: Current engine torque.
 
+
+
+#
+# Diesel Particulate Filter
 #
-# Current Ambient (Outside) Air Temperature
+- DieselParticulateFilter:
```

Comment:
I am not sure, if we better have an Exhaust branch. Because thee wil be stuff such as SCR tank and injection and NoX values